Formula/slirp4netns.rb

```diff
@@ -38,6 +38,7 @@ def install
   test do
     resource("test-common").stage (testpath/"test")
     resource("test-api-socket").stage (testpath/"test")
-    assert_match /unshare\: unshare failed: Operation not permitted/, shell_output("bash ./test/test-slirp4netns-api-socket.sh", 1)
+    output = shell_output("bash ./test/test-slirp4netns-api-socket.sh", 1)
+    assert_match /unshare: unshare failed: Operation not permitted/, output
```

Comment:
```suggestion
    assert_match "unshare: unshare failed: Operation not permitted", output
```
You don't need a regexp here.

Reply:
Thanks for the suggestion, but I am not sure whether the error message is from `stdout` or `stderr`. I may need to trigger the CI again to try it out.